main.c

```diff
@@ -81,6 +81,7 @@ int main() {
 				if (userEnterAction) {
 					clear();
 					gameState = GAME_INTRODUCTION_STATE;
+					timeSinceLastGameBoardDecrease = clock();
```

Comment:
Why you are setting the _timeSinceLastGameBoardDecrease_ at the main init commands and here? I think that set it here is enough.

Reply:
If we do not reset the value of that variable to clock() in case the player win or lose, the program will use the last value of time used.

Reply:
Ok!